app/chitietdonhang.jsx

```
@@ -0,0 +1,406 @@
+                activeOpacity={0.8}
+              >
+                <Feather name="star" size={16} color="#fff" />
+                <Text
+                  style={{
+                    color: '#fff',
+                    fontWeight: '600',
+                    fontSize: 13,
+                    marginLeft: 6,
+                  }}
+                >
+                  Đánh giá "
```

ESLint

[error] `"` can be escaped with `&quot;`, `&ldquo;`, `&#34;`, `&rdquo;`.

(react/no-unescaped-entities)